INSTALL.md

```diff
@@ -210,3 +212,34 @@ You can use the [conda package manager](https://conda.io/) that comes with the
    ```
    conda install gmt -c conda-forge/label/dev
    ```
+
+## FreeBSD
+
+GMT may be installed on FreeBSD using Ports or from source.
+
+**NOTE:** The Ports Collection may provide old GMT versions. If you want the latest GMT release, consider [building the
+latest release from source](BUILDING.md).
+
+### Install via Ports
+
+The FreeBSD Ports Collection is a diverse collection of utility and application software that has been ported to FreeBSD.
+
+If not done already, set up the **Ports Collection** (see
+https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/ports-using.html):
+
+```
+$ portsnap fetch
+$ portsnap extract
```

Comment:
We don't add the `$` sign for other install instructions. It's better to keep it consistent.
```suggestion
portsnap fetch
portsnap extract
```